api/prisma.config.ts

```diff
@@ -0,0 +1,6 @@
+import path from 'node:path';
+import type { PrismaConfig } from 'prisma';
+
+export default {
+  schema: path.join('prisma')
```

Comment:
```suggestion
  schema: 'prisma'
```
I know it's in the docs, but joining one thing does nothing.